src/run.py

```diff
@@ -0,0 +1,256 @@
+        print("queryByNationality..")
+        dataUtility=DataUtility(self.config['dbName'])
+        cond=''
+        if(nationality!='' and nationality!='all'):
+            cond='WHERE NATIONALITY=?'
+        datas=dataUtility.query(cond,nationality.replace('"',''))
+        print("####Result####")
+        for data in datas:
+            print(f'{data[0],data[1],data[2],data[3],data[4],data[5],data[6],data[7],data[8],data[9],data[10],data[11]}')
+
+    @usedtime
+    def queryByDepartment(self,department):
+        print("queryByDepartment..")
+        dataUtility=DataUtility(self.config['dbName'])
+        cond=''
+        if(department!='' and department!='all'):
+            cond='WHERE DEPT=?'
+        datas=dataUtility.query(cond,department.replace('"',''))
+        print("####Result####")
+        for data in datas:
+            print(f'{data[0],data[1],data[2],data[3],data[4],data[5],data[6],data[7],data[8],data[9],data[10],data[11]}')
+
+def printInfo():
+    print("- arg1 Execute type\n  - M Migrate\n- arg2 Datasource file\n- arg3 Target db\n- arg4 Report name")
+    print("Example\n python runv2.py M ../data-devclub-1.xml ../database/devclub2022.db ../csv ../reports/data-devclub-report.json")
+def printInfoQ():
+    print("- arg1 Execute type\n  - M Migrate\n- arg2 Target db file\n- arg3 Search Type\n- arg4 Value Search")
+    print("Example\n python run.py Q ../database/devclub2022.db region all")
+    print('Example\n python run.py Q ../database/devclub2022.db region "Canada"')
+    print("Example\n python run.py Q ../database/devclub2022.db dept all")
+    print('Example\n python run.py Q ../database/devclub2022.db dept "Flight Planning"')
+    print("Example\n python run.py Q ../database/devclub2022.db nationality all")
+    print('Example\n python run.py Q ../database/devclub2022.db nationality "Germany"')
+def main():
+    try:
+
+        print("#####start#####")
+        print(f"{sys.argv}")
+        if(len(sys.argv)>1):
+            exeType = sys.argv[1]
+            if (exeType=='M'):
+                datasource = sys.argv[2]
+                dbName = sys.argv[3]
+                csvPath = sys.argv[4]
+                reportName = sys.argv[5]
+                config={
+                    "datasource":datasource,
+                    "dbName":dbName,
+                    "csvPath":csvPath,
+                    "clubDataReport":reportName
+                }
+
+                startTime = time.perf_counter()
+                        
+                exe=Executor()
+                exe.setup(config)
+                exe.extract()
+                exe.transform()
+                exe.generateCSVByNationality()
+                # exe.load()
+                exe.loadFromCSV()
+                exe.generateSummary()
+                
+
+                endTime = time.perf_counter()
+                totalTime = endTime - startTime
+                print(f'##Total used time {totalTime:.4f} seconds##')
```

Comment:
instead of long code block condition I think you can wrap up with function it make more readable